install.sh

```diff
@@ -194,9 +194,6 @@ info  "-y (--yes-to-all):       ${arg_y}"
 this_script="$(basename "$0")"
 export this_script
 
-export install_path="${arg_i%/}"
-info "install_path=\"${install_path}\""
-
```

Comment:
I'll just re-insert the lines.  I'd like to get this into a release ASAP and then point people on comp.lang.fortran to the release in case anyone wishes to install the current gfortran 7.0.0 trunk to try out user-defined derived type input/output (UD-DTIO), which two gfortran developers just committed to the trunk based in part on promised funding from Sourcery Institute.